util/venv-dependencies.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
Can you add a top-level comment on this file similar to the ubuntu dep script that describes its purpose & when/where you would want to use it?

Reply:
Done.